modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewPage.kt

```diff
@@ -156,7 +156,11 @@ private fun WhatsNewPageLoaded(
                 modifier = Modifier.padding(horizontal = 32.dp),
             )
 
-            Spacer(modifier = Modifier.height(32.dp))
+            if (state.fullModel) {
```

Comment:
I added this code back in order to put the tittle in center as we were doing before sharing clip whats new